elements/pfe-tabs/test/pfe-tabs_e2e.js

```diff
@@ -1,15 +1,24 @@
 const element = require("../package.json").pfelement.elementName;
 
 describe(element, () => {
+  let snapshotOptions;
+
   before(() => {
     browser.url(`/elements/${element}/demo`);
-  });
 
-  it("should take a screenshot", () => {
-    browser.saveFullPageScreen(element);
+    snapshotOptions = {
+      removeElements: [
+        $("#toc"),
+        $$(".skip-e2e")
+      ]
+    };
   });
 
-  it("should compare to the baseline", () => {
-    expect(browser.checkFullPageScreen(element)).toBeLessThan(3.1);
+  ["light", "dark", "saturated"].forEach(context => {
```

Comment:
This is elegant! Do you think we lose any development benefits of seeing all the variants on one page? What if we leave the simplified demo/index.html as you have it, and then we add in a new demo/all-combos.html or something, where you can see everything in a glance? @castastrophe

Reply:
The challenge is that you can't see it at a glance because the page is so tall. You have to download it and zoom in to try to compare. It gets tricky. I was hoping separating them into individual files makes it easier to diff.

Reply:
For sure, I'm just highlighting the different between AT and development. I'm in favor of this smaller version for AT and VRT, but as a dev working on tabs which have a lot of variants, its nice to not have to recreate all the combos.

Reply:
I get that but currently there is no difference between AT and the demo pages so we'll have to find a happy in-between.

Reply:
pfe-styles has multiple demo files, couldn't this component have a couple too? and just use the index.html as the one AT uses?

Reply:
AT fails on those other URLs more often.  No idea why but when we've tried to snapshot other pages, it fails more often than passes due to 404 errors.  It will all get a lot easier after Zack's work but for now, we work with what we've got.

Reply:
I'm suggesting 
1. `demo/index.html` - use the exact copy you have in this PR, and run AT on this file only
2. `demo/all-combos.html` - devs can navigate to this page and use it during development. The old version of the demo markup lives here. AT does not run on this file

Reply:
Yes! Great point. I hadn't considered that. I'll do that first thing tomorrow!